espressif/idf-eclipse-plugin
# WIP: IEP:688 Visualizing sensor data

## Description

Simple telemetry viewer, which allows visualizing data from the serial monitor.

Fixes # ([IEP-688](https://jira.espressif.com:8443/browse/IEP-688))

## Type of change

Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)

## How has this been tested?

Test 1:
- create a simple project and imutate data from the sensors with the printf() function (or use real sensors, potentiometer, etc..) 
- open the telemtry dialog by clicking on Telemetry Icon (The icon will be changed in future)
![Telemetry_Icon](https://user-images.githubusercontent.com/24419842/219328072-6550df2d-a3fc-4b6d-808e-2ad1c48f12ff.png)
- specify the data format and other parameters
- open the serial monitor
- Graph should be updated after serial monitor is opened and data is read and printed to the serial monitor

**Test Configuration**:
* ESP-IDF Version:
* OS (Windows,Linux and macOS):

## Dependent components impacted by this PR:

- Telemetry Viewer
- Serial Monitor

## Checklist
